skyportal/handlers/api/public_pages/public_source_page.py

```diff
@@ -22,19 +22,20 @@ def calculate_hash(data):
 
 
 def process_thumbnails(thumbnails, ra, dec):
+    has_ls = any("ls" in thumbnail["type"] for thumbnail in thumbnails)
```

Comment:
Can you add some comments on what this does? I guess the "dr8" is specifically to avoid showing in favor of dr9?